class-tgm-plugin-activation.php

```diff
@@ -1128,6 +1101,50 @@ public function _get_plugin_data_from_name( $name, $data = 'slug' ) {
 		}
 
 		/**
+		 * Retrieve a link to a plugin information page
+		 *
+		 * @since 2.5.0
+		 *
+		 * @param string $slug
+		 *
+		 * @return string
+		 */
+		public function get_plugin_info_link( $slug ) {
```

Comment:
Why is this one named with "plugin", yet the `get_download_link()` method isn't? Both are talking about plugins, so some consistency either way would be nice.